akka-typed/src/test/scala/akka/typed/ActorContextSpec.scala

```diff
@@ -141,36 +140,33 @@ object ActorContextSpec {
           }
         case BecomeCareless(replyTo) ⇒
           replyTo ! BecameCareless
-          Actor.Stateful(
-            (ctx, message) ⇒ Actor.Unhandled,
-            (ctx, signal) ⇒ signal match {
-              case Terminated(_) ⇒ Actor.Unhandled
-              case sig ⇒
-                monitor ! GotSignal(sig)
-                Actor.Same
-            })
+          Actor.Stateful[Command]({
+            case (_, _) ⇒ Actor.Unhandled
```

Comment:
I'll leave that out of this PR as a future improvement, it will be for the respective dsls because the ActorContext have different DSL specific types, and there already is a few such private methods over there.